assets/images/people/avatar_dummy.svg

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Can you add a `avatar_dummy.svg.source` file that links back to the source?

Reply:
... containing just the link ?